docs/troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+# Common troubleshooting tips
+
+## Module not found errors
+
+This is the most common error rules_js users encounter.
+These problems generally stem from a runtime `require` call of some library which was not declared as a dependency.
+
+Fortunately, these problems are not unique to Bazel.
+As described in https://github.com/aspect-build/rules_js/blob/main/docs/pnpm.md#hoisting, rules_js
```

Comment:
Link right to `docs.aspect.build` docsite equivalent from here?

Reply:
this should just be a relative link so you stay on whichever site you're on, fixed